arelle/PluginManager.py

```diff
@@ -357,6 +362,29 @@ def _get_name_dir_prefix(
 
     return (None, None, None)
 
+def _get_location(moduleDir: str, moduleName: str) -> str:
+    """Get the file name of a plugin."""
+    if os.path.isfile(Path(f"{moduleDir}/{moduleName}.py")):
+        return Path(f"{moduleDir}/{moduleName}.py")
```

Comment:
nit: define this path once as a variable and reuse it between the `return` statement and `isfile` conditional.

Reply:
Just pushed this change